bundles/org.dataflowanalysis.analysis.converter/src/org/dataflowanalysis/analysis/converter/DataFlowDiagramConverter.java

```diff
@@ -0,0 +1,432 @@
+                mapPinToAssignments.get(assignment.getOutputPin()).add(assignment);
+            } else {
+                List<AbstractAssignment> list = new ArrayList<>();
+                list.add(assignment);
+                mapPinToAssignments.put(assignment.getOutputPin(), list);
+            }
+        }
+        return mapPinToAssignments;
+    }
+
+    private String createBehaviourString(List<AbstractAssignment> abstractAssignments) {
+        StringBuilder builder = new StringBuilder();
+        if (abstractAssignments != null) {
+            for (AbstractAssignment abstractAssignment : abstractAssignments) {
+                if (abstractAssignment instanceof ForwardingAssignment) {
+                    for (Pin inPin : abstractAssignment.getInputPins()) {
+                        builder.append("forward ").append(mapInputPinToFlowName.get(inPin)).append("\n");
+                    }
+                } else {
+                    Assignment assignment = (Assignment) abstractAssignment;
+                    String value = getTermValue(assignment.getTerm()) ? "TRUE" : "FALSE";
+
+                    for (Label label : assignment.getOutputLabels()) {
+                        try {
+                            builder.append("set ").append(((LabelType) label.eContainer()).getEntityName()).append(".").append(label.getEntityName())
+                                    .append(" = ").append(value).append("\n");
+                        } catch (IllegalArgumentException ex) {
+                            System.out.println(
+                                    "Caution!! WebEditor cant handle complex Assignments yet. Only TRUE or NOT(TRUE) supported. Everything else ignored");
+                        }
+                    }
+                }
+            }
+            return builder.toString().trim();
+        }
+        return null;
+
+    }
+
+    // Currently only supports True or False
+    private boolean getTermValue(Term term) throws IllegalArgumentException {
+        if (term instanceof TRUE)
+            return true;
+        if (term instanceof NOT) {
+            return !getTermValue(((NOT) term).getNegatedTerm());
+        }
+        throw new IllegalArgumentException();
+    }
+
+    private void parseBehavior(Node node, Pin outpin, String lines, DataFlowDiagram dfd, DataDictionary dd) {
+        String[] behaviorStrings = lines.split("\n");
+        var behavior = node.getBehaviour();
+        for (String behaviorString : behaviorStrings) {
+            if (behaviorString.contains("forward ")) {
+                String packet = behaviorString.split(" ")[1];
+                Pin inpin = null;
+                for (Flow flow : dfd.getFlows()) {
+                    if (flow.getDestinationNode() == node) {
+                        if (flow.getEntityName().equals(packet)) {
+                            inpin = flow.getDestinationPin();
+                        }
+                    }
+                }
```

Comment:
Can be replaced by a stream expression and `findAny()`. Furthermore, error handling is missing